tools/db/build/create-database.sql

```diff
@@ -1,11 +1,20 @@
 -- We'll go ahead and create databases if they are not present, before choosing the database to populate based on the t_active setting
 
-IF DB_ID('keyboards') IS NULL
-BEGIN
-  CREATE DATABASE keyboards;
-END
+USE master;
+GO
 
-IF DB_ID('keyboards_1') IS NULL
+IF DB_ID('$keyboards') IS NULL
```

Comment:
`$keyboards` is replaced by the `sqlrun()` function with the name of the database specified in environment.